src/renderer/api/endpoints/ingress.api.ts

```diff
@@ -2,6 +2,7 @@ import { KubeObject } from "../kube-object";
 import { autobind } from "../../utils";
 import { IMetrics, metricsApi } from "./metrics.api";
 import { KubeApi } from "../kube-api";
+import { hostname } from "os";
```

Comment:
Is this used anywhere?

Reply:
Good catch. Have removed in the next commit